GoogleForCreators/web-stories-wp
# Design System: KeyboardOnlyOutline

## Summary

Adding existing keyboardOnlyOutline global style that is in play on editor and dashboard to design system. This will allow us to remove the util in editor and dashboard.

## Relevant Technical Choices

Setting up KeyboardOnlyOutline in storybook preview for design system rendering. 
Removing excess outline: none on components of design system (button, dropDown, pill, typography) and relying on theme/helpers/outline. 
~`<KeyboardOnlyOutline />` can be accessed in apps through `theme/index` by importing `ThemeGlobals.KeyboardOnlyOutline`. This will set us up to have other globals all stored together in theme, while their names can preserve context.~
Swapped existing `KeyboardOnlyOutline` for new `OverrideFocusOutline` that uses [`focus-visible`](https://github.com/WICG/focus-visible) to polyfill and allow us to take advantage of `:focus-visible` (https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo). This means we can get away from the custom hook to set up when to have focus styles show up.  

## To-do

N/A

## User-facing changes

None, storybook only

## Testing Instructions

Spin up storybook and go to design system. Click on a button, see that the blue border doesn't show up. Now tab onto it and also hit enter, see that the blue border is present. 

---



Addresses #5810
